src/engraving/libmscore/system.h

```diff
@@ -154,7 +158,7 @@ class System final : public EngravingItem
 
     Page* page() const { return (Page*)parent(); }
 
-    void layoutSystem(const mu::engraving::LayoutContext& ctx, qreal, const bool isFirstSystem = false, bool firstSystemIndent = false);
+    void layoutSystem(const mu::engraving::LayoutContext& ctx, qreal xo1, const bool isFirstSystem = false, bool firstSystemIndent = false);
```

Comment:
this chenge looks superfluous

Reply:
Yeah, it's added for consistency. Figured since I was updated this part of the code in the `cpp` file, this was probably the most relevant time to improve the code health slightly.

I can undo this if it's superfluous and should be removed.

Reply:
Nah, is is fine.